python/llm/example/CPU/PyTorch-Models/More-Data-Types/README.md

```diff
@@ -0,0 +1,41 @@
+# BigDL-LLM More Data Types for Low-Bit Inference on Large Language Model
+
+In this example, we show how to apply BigDL-LLM low-bit optimizations (including INT8/INT5/INT4) to Llama2 model, and then run inference on the optimized low-bit model.
+
+## 0. Requirements
+To run this example with BigDL-LLM, we have some recommended requirements for your machine, please refer to [here](../../README.md#system-support) for more information.
+
+## Example: Predict Tokens using `generate()` API
+In the example [generate.py](./generate.py), we show a basic use case of low-bit optimizations on a Llama2 model to predict the next N tokens using `generate()` API, with BigDL-LLM low-bit optimizations.
```

Comment:
I think we should provide more descriptions about the usage of applying other low-bit optimization at the beginning.

Reply:
Done.